Sources/Controllers/MessagesViewController.swift

```diff
@@ -328,3 +366,35 @@ fileprivate extension MessagesViewController {
         return 0
     }
 }
+
+// MARK: - Menu Handling
+extension MessagesViewController {
+    /// Add observer for `UIMenuControllerWillShowMenu` notification
+    fileprivate func addMenuControllerObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(MessagesViewController.menuControllerWillShow(aNotification:)), name: NSNotification.Name.UIMenuControllerWillShowMenu, object: nil)
+    }
+    
+    /// Remove observer for `UIMenuControllerWillShowMenu` notification
+    fileprivate func removeMenuControllerObservers() {
+        NotificationCenter.default.removeObserver(self, name: NSNotification.Name.UIMenuControllerWillShowMenu, object: nil)
+    }
+    
+    /// Show menuController and set target rect to selected bubble
+    @objc func menuControllerWillShow(aNotification:Notification) {
```

Comment:
@zhongwuzw Can we just name this parameter `notification`? And add a space after the colon